deel-3-webpagina/styles/styles.css

```diff
@@ -1 +1,64 @@
-/*Schrijf hier jouw css*/
\ No newline at end of file
+/*Schrijf hier jouw css*/
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body {
+    display: flex;
+    flex-direction: column;
+    background-color: #F2F3D9;
+    font-family: Poppins, Arial, sans-serif;
+    color: #030027;
+    text-align: center;
+    min-height: 100vh;
+}
+.header-content {
```

Comment:
het werkt. Alleen de vraag is of je hier een flex-box nodig hebt. In de NOVI uitwerking maken ze van die h1 en h3 titels geen flex-box. Dat komt pas bij de 3 articles. 
NOVI heeft alleen  ="about-us-section"  section met dus die 2 titels, dan een inner-container met de articles. Dan alleen padding nodig voor deze twee titels h1 en h3.  Jij hebt een section-content met een header content  en content.